pyproject.toml

```
@@ -39,7 +39,11 @@ classifiers = [
   "Topic :: Software Development :: Libraries :: Python Modules",
   "Topic :: Software Development :: Quality Assurance",
 ]
-urls = { repository = "https://github.com/charliermarsh/ruff" }
+
+[project.urls]
+Repository = "https://github.com/charliermarsh/ruff"
+Documentation = "https://beta.ruff.rs/docs/"
+Changelog = "https://github.com/charliermarsh/ruff/releases"
```

```suggestion
repository = "https://github.com/charliermarsh/ruff"
documentation = "https://beta.ruff.rs/docs/"
changelog = "https://github.com/charliermarsh/ruff/releases"
```

not sure how picky different tools are, but in the [docs](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#urls) they always use lowercase keys in toml

---

I think those docs are wrong, I have opened a PR against it, see:

* https://github.com/pypa/packaging.python.org/pull/1250

---

sorry i only say your comment after looking at the code, you're likely right about this

---

But I know that Renovate bot does a case-insensitive comparison, so it doesn't matter for Renovate.

---

fwiw warehouse also does case insensitive checks: https://github.com/pypi/warehouse/blob/9cb091572194a3881bd8ba5164cf78b5a87bc5df/warehouse/templates/packaging/detail.html#L20-L31